docs/src/admin-manual/configuration.md

```diff
@@ -0,0 +1,1292 @@
+      installed. Note that the project's Docker configuration automatically
+      installs this dependency in the Enduro Docker image.
+    * The file **MUST** be named `premis.xml`
+    * The file **MUST** be found in a `metadata` directory
+    * The metadata directory **must not** be nested inside any other directories
+    * The PREMIS schema file path must be accessible to Enduro
+
+**Default values**:
+
+```toml
+[validatePremis]
+enabled = true
+xsdPath = "/home/enduro/premis.xsd"
+```
+
+* `enabled`: Set to `true` by default. To skip PREMIS validation, change this
+  value to `false`.
+* `xsdPath`: Defines the filesystem path where Enduro can find the PREMIS 3
+  XML Schema Definition file to be used for validation.
+
+    Enduro does include a PREMIS 3.0 XSD file at installation, located at
+    `hack/xsd/premis.xsd` from the root Enduro installation directory.
+
+### Watched location configuration
+
+These configuration settings, when enabled, allow Enduro to initiate SIP ingest
+from a watched location. The configured watched location can either be a local
+filesystem directory or an object store bucket (such as one provided by MinIO,
+S3, or Azure). Once configured, any time a new zipped package is added to the
+location, Enduro's watcher (in this case, [Redis]) will see it and automatically
+initiate an ingest workflow. For more information, see:
+
+* [Initiate ingest via a watched location upload][watched-location]
+
+At this time, [Redis] is the only supported watcher - as such, the
+`redisAddress` and `redisList` fields are required.
+
+All other default parameters are S3-specific, and might not be needed for other
+watched location types. The default configuration included uses a [MinIO] bucket
+as the example watched location. MinIO uses Amazon [S3] syntax for its
+configuration properties. Different object stores may have different parameters
+to be configured. Consult the corresponding object store provider's
+documentation for more information.
+
+For **local filesystems**, the only other required field is a `url` parameter
+pointing to the target location - for example:
+
+```toml
+url = "file:///home/enduro/watchdir"
+```
+
+Note that when configuring a local filesystem directory as a watched location,
+a watcher must also be installed and configured - simply using the default
+[Redis] values included below will **not work without further set-up**.
+
+**Default values**:
+
+```toml
+[watcher.embedded]
+name = "dev-minio"
+redisAddress = "redis://redis.enduro-sdps:6379"
+redisList = "minio-events"
+endpoint = "http://minio.enduro-sdps:9000"
+pathStyle = true
+key = "minio"
+secret = "minio123"
+region = "us-west-1"
+bucket = "sips"
+workflowType = "create aip"
+```
+
+* `name`: Defines a name to be used internally for the watched location. Useful
+  if you are configuring more than one watched location.
+* `redisAddress`: Binds Redis to a specific address and port.
+* `redisList`:  = "minio-events"
```

Comment:
Not sure what happened here, but this should be a description I think?

Reply:
oof, cut/paste error, where i missed updating it after starting w the default values.  I have updated this to:

> * `redisList`: The name of the queue that Redis should use for the watched location SIP deposit events.